test/service/actions/users.test.js

```diff
@@ -130,7 +130,7 @@ describe('Actions.Users', () => {
                 }
             });
 
-            Actions.getProfiles(0)(store.dispatch, store.getState);
+            Actions.getProfiles(0, 500)(store.dispatch, store.getState);
```

Comment:
Perhaps we shouldn't be checking that we got back that specific profile in this call. That would mirror how the api tests on the server work.